test_sb/test_sb.py

```diff
@@ -0,0 +1,21 @@
+from bit_vector import BitVector
+from sb.sb import gen_sb
+
+
+def test_sb_functional_model():
+    """
+    Small tests to show the usage of the sb functional model.
+    """
+    sb = gen_sb(16, 2, 4, "00", "11", 1, 0, 101010)
+    sb_inst = sb()
+    sb_inst.reset()
+    sb_inst.configure(BitVector(0, 32), BitVector(65535, 32))
+    data = [11, 12, 21, 22, 31, 32, 41, 42, 46]
```

Comment:
can this data be a random vector?

Can you add some comments about why the output data is what you assert it to be? Similarly for why the config values are what you've chosen...basically we want to have some high level knowledge of how to use the model.

Alternatively, we can leave this as the first pass and then iterate to build a higher level model next (e.g. where we configure registers directly).

Reply:
I have updated the comments in the file. For the data vector, the 1st digit is side value and 2nd digit is track value. So it's easy to debug the output based off the configuration input. The two tests I use for the functional model are the same as what I did in the verilog TB to ensure RTL and model behavior are the same. Once the driver is in place to do direct comparison between RTL & model output, the tests should definitely be extended for a wider range of input configuration values.

Reply:
Updated the note and also added a new random data input test.